examples/TestDummy/directive-with-dummy.js

```diff
@@ -26,27 +26,22 @@ angular
 
 
 describe('someDirective', function() {
-  var
-    element, $compile, $rootScope, $scope;
-
-  beforeEach(module('directiveWithDummy', {
-    anotherElementDirective: TestDummy.directive
-  }));
-
-  beforeEach(function() {
-    inject(function(_$compile_, _$rootScope_) {
-      $compile = _$compile_;
-      $rootScope = _$rootScope_;
-      $scope = $rootScope;
-      element = angular.element('<some-directive></some-directive>');
-      $compile(element)($scope);
-      angular.element(document.body).append(element);
-      $scope.$apply();
-    });
+  var Test, element, $scope;
+
+  module.sharedInjector();
```

Comment:
Angular.mock.module.sharedInjector is available from version 1.5.1 (https://code.angularjs.org/1.5.1/docs/api/ngMock/function/angular.mock.module.sharedInjector). You should add somewhere comment with this information.